oxidation/client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "npm run wasm:build && npm run wasm:copy && webpack && vue-cli-service serve",
```

Comment:
why calling `webpack` directly is needed here ? I would supposed `vue-cli-service` has it own way to do that (otherwise there is no hot reload ? or hot reload only works when modifying the project's source code ?)

Reply:
Good catch :thinking: vue-cli do the same thing